lib/src/commands/packages/commands/check/commands/licenses.dart

```diff
@@ -241,7 +241,7 @@ class PackagesCheckLicensesCommand extends Command<int> {
         continue;
       }
 
-      final packagePath = path.normalize(cachePackageEntry.root.path);
+      final packagePath = path.normalize(path.fromUri(cachePackageEntry.root));
```

Comment:
We should try to run our e2e test on windows because it would most likely have been caught then: https://github.com/VeryGoodOpenSource/very_good_cli/blob/main/.github/workflows/e2e.yaml#L51